safe-global/web-core
# feat: new creation flow design + first step

## What it solves

Resolves #961

## How this PR fixes it

This implements the new creation/load flow layout, as well as the first step of creation. The progress bar is a placeholder as it will rely on all steps/progress and the stepper hook.

The widgets are not yet included as the overview one requires form data and the info one is yet to be QA'd.

Note: I've removed all legacy creation code, apart from `sender.ts` and `useEstimateSafeCreationGas` (and it's test). The shared `types.d.ts` and `OwnerRow.tsx` have been moved into `/load-safe` for now.

## How to test it

Attempt to create a new Safe and observe the new layout. Clicking submit show log the form values (with the default Safe name if none is entered). It should also react well to dark mode.

## Screenshots

![creation](https://user-images.githubusercontent.com/20442784/197543410-47b03e38-d603-4957-bbc4-8a39bb9413e5.gif)
